app/build.gradle

```diff
@@ -20,7 +20,11 @@ android {
         abortOnError false
     }
 
-    buildTypes {
+    productFlavors {
+        travis{}
```

Comment:
In this case it creates a product flavor, travis.  Product flavors are basically maven build profiles but for Android.  In this case it lets us supply a different google-services.json file to use when we are running the travis build.